src/GNDStk/child_t/src/operator.hpp

```diff
@@ -16,6 +16,42 @@ inline auto operator-(const child_t<TYPE,ALLOW,CONVERTER,FILTER> &kwd)
 
 
 
+// -----------------------------------------------------------------------------
+// operator~: Allow as top-level
+// operator!: Disallow as top-level
+//
+// We don't expect much use of these, but someone may occasionally find the
+// first (more likely) or the second (less likely) to be helpful. The unary
+// operators that we chose for these purposes seemed like the best, given a
+// limited selection. One could think: (T)ilde for (T)op; Not for Not.
+// -----------------------------------------------------------------------------
+
+// operator~
+template<class TYPE, allow ALLOW, class CONVERTER, class FILTER>
+inline auto operator~(const child_t<TYPE,ALLOW,CONVERTER,FILTER> &kwd)
```

Comment:
While I like operator overloading, "assigning" a different meaning can be dangerous. Again, good documentation is required. I was wondering here if this couldn't give issues with destructors?

Why not use an actual function name instead of an operator overload?

Reply:
I'll make a note to consider this for the next change. The "top-level" business needs some work, anyway, and isn't likely to be something that someone would need to change. So, perhaps I will just make these into regular functions. I didn't actually even think about the destructor issue when I was writing this, but I think the difference is that tilde as the destructor is a member function, while this one isn't, so they're unambiguous.